src/helpers/response-status.helper.ts

```diff
@@ -0,0 +1,36 @@
+import { HttpStatus } from '@nestjs/common';
+import { isAxiosError } from 'axios';
+
+/**
+ * Checks if the error object is an AxiosError and has a response status matching the specified status.
+ * @param {Object} options - An object containing the error and status.
```

Comment:
We shouldn't comment types here as it overrides and mucks around with typescript -- It's good to list the params name with a description if it helps tho :)

Reply:
Looks like recommendations exist to not use jsDoc types in Typescript.

Google style guide - https://google.github.io/styleguide/tsguide.html#jsdoc-type-annotations
```JSDoc type annotations are redundant in TypeScript source code. Do not declare types in @param or @return blocks, do not write @implements, @enum, @private, @override etc. on code that uses the implements, enum, private, override etc. keywords.```

So you could remove {Object}, {Error}, {HttpStatus}, {boolean}

Reply:
Have removed.